pkg/apis/authorization/types.go

```diff
@@ -103,9 +111,13 @@ type SubjectAccessReviewSpec struct {
 	Groups []string
 	// Extra corresponds to the user.Info.GetExtra() method from the authenticator.  Since that is input to the authorizer
 	// it needs a reflection here.
-	Extra map[string][]string
+	Extra map[string]ExtraValue
```

Comment:
This doesn't change any external serialization, but it's required so that protobuf generates the nested types properly.